### PR TITLE
Remote transform initial pass

### LINKED ISSUE #94: RFC: `-t` for remote codemods
Currently, community-specific codemods are appearing inside specific repos without much shareability and the ergonomics to apply them require quite a bit of manual effort (especially for a tool based around removing manual effort).

Allowing `-t` to accept remote codemods would enable a generic way to build a hosted service and share codemods.

### PR DESCRIPTION
**Pull request by iamdustan**
An initial pass at including remote support for the transform flag.
```sh
jscodeshift -t https://raw.githubusercontent.com/reactjs/react-codemod/master/transforms/react-to-react-dom.js .
```

One constraint is the remote transform must be entirely self-contained.

- :white_check_mark: https://raw.githubusercontent.com/reactjs/react-codemod/raw/master/transforms/react-to-react-dom.js
- :x: https://raw.githubusercontent.com/reactjs/react-codemod/raw/master/transforms/create-element-to-jsx.js

Closes #94.

### CODE REVIEW

File: src/Runner.js [hunk 1]
```diff
@@ -120,102 +123,141 @@ function run(transformFile, paths, options) {
   const statsCounter = {};
   const startTime = process.hrtime();
 
-  if (!fs.existsSync(transformFile)) {
+  if (/^http/.test(transformFile)) {
+    if (transformFile.indexOf('https') !== 0) {
+      console.log(
+        clc.whiteBright.bgRed('ERROR') + ' Remote transforms only support https. %s',
```
Comment (fkling) [inline src/Runner.js]: Why is that?

Comment (iamdustan) [inline src/Runner.js]: No real reason. Maybe security?

Comment (iamdustan) [inline src/Runner.js]: removed this constraint

**Comment by fkling** [comment]
Merged with 027cd027800e238e35d047654360ba08d8e3f7ea. The merge conflict was a a bit tricky to solve, I hope I did everything correctly :P (test still pass!)

**Comment by fkling** [comment]
Released with v0.3.15.

**Pull request closed without merging**